pkg/backup/backup_new_test.go

```diff
@@ -827,9 +826,13 @@ func TestBackupActionsRunForCorrectItems(t *testing.T) {
 					newPV("pv-1"),
 					newPV("pv-2"),
 				),
+				namespaces(
+					newNamespace("ns-1"),
+					newNamespace("ns-2"),
+				),
 			},
 			actions: map[*recordResourcesAction][]string{
-				new(recordResourcesAction).ForNamespace("ns-1"): {"ns-1/pod-1", "ns-1/pvc-1", "pv-1", "pv-2"},
+				new(recordResourcesAction).ForNamespace("ns-1"): {"ns-1/pod-1", "ns-1/pvc-1", "ns-1", "ns-2"},
```

Comment:
@skriss not sure if I got this right that all Namespace resources should have actions run, seems odd that actions should run for "ns-2"?

Reply:
Yeah, I was just looking at that. TBH I think it's probably more logical to *not* run these actions for Namespace resources themselves, only for the namespaced resources in the specified namespace(s).

Reply:
@nrb thoughts?

Reply:
oops - just updated to not run these for namespace resources, but happy to change again based on @nrb's feedback

Reply:
I think what you have now makes sense, but we'll get @nrb's opinion too.

Reply:
I left the namespace resources in the test as well to make this more explicit, but can remove if we don't think it's really necessary

Reply:
I'd keep it, adds some clarity.